chain/src/main/java/org/ethereum/beacon/chain/storage/ChainStorage.java

```diff
@@ -0,0 +1,17 @@
+package org.ethereum.beacon.chain.storage;
+
+import org.ethereum.beacon.db.Database;
+
+/**
+ * Created by Anton Nashatyrev on 27.12.2018.
+ */
+public interface ChainStorage {
```

Comment:
Doesn't look like something abstract.
Though I'm ok to keep naming simple if it's in specific package.

Reply:
Not sure what you mean here... Let's talk on tomorrow call

Reply:
I mean `ChainStorage` sounds abstract but `BeaconBlockStorage` etc are very specific.  So smth like `BeaconChainStorage` and `AbstractBeaconChainStorage` could be better.
But anyway I'm ok to keep it shorter like it is now.